case_studies/san_leandro_high_school/01_club_fair/post_mortem.md

```diff
@@ -0,0 +1,25 @@
+# Post-mortem
+
+##### What We Did:
+  * Encouraged people to sign up.
+  * Passed out flyers.
+  * Explained our mission.
+
+#### How It Differed from Our Plan:
+  * More people signed up than expected.
+  * We had to do more explaining than we thought.
+
+##### What Went Well:
+  * Many people signed up.
+  * People seemed to be interested about Hack Club.
+  * We were able to talk to people with a wide range of programming knowledge and ability.
+  
+##### What Didn't Go Well:
+  * Some people misunderstood the type hacking we did; they asked if we could teach them how to DDOS.
+  * There was confusion as to when and where the meetings were.
+
```

Comment:
Same comments for this section as in https://github.com/hackedu/hackedu/pull/206/files#r39449571.